client/src/pages/Products/Products.jsx

```diff
@@ -74,7 +74,7 @@ const Products = () => {
       try {
         setLoading(true);
         const res = await axios.get(
-          `${import.meta.env.VITE_SERVER_URL}/api/v1/product/filtered-products`,
+          `https://e-commerce-mgtd.onrender.com/api/v1/product/filtered-products`,
```

Comment:
Don't hard code the server base url as it'll be difficult for other developer to contribute to the project. In case you want to use deployed server just update the server url in .env file with-> https://e-commerce-mgtd.onrender.com without any / at end of it.